### PR TITLE
Add location message to message types collection

### PR DESCRIPTION
**Pull request by TomFellows**
This PR adds a `POST` `Location Message` to the public collection. The collection in `dev-tools` already has a `location message` collection entry.

JIRA: https://zendesk.atlassian.net/browse/SCSDK-88?atlOrigin=eyJpIjoiMmM4NDMzOWMwY2NlNDZmMjk4ZTM4YzIwZjhkNmY4NzQiLCJwIjoiaiJ9

### CODE REVIEW

**Review by jugarrit — APPROVED** [review]
We store responses? 🤔

**Comment by TomFellows** [comment]
> We store responses? 🤔

The other message types had examples of expected responses. Maybe it wasn't necessary 😅

File: Smooch.postman_collection.json [hunk 1]
```diff
@@ -10632,6 +10632,180 @@
...
+								},
+								{
+									"key": "X-XSS-Protection",
+									"value": "1; mode=block",
+									"name": "X-XSS-Protection",
+									"description": "Cross-site scripting (XSS) filter"
+								}
+							],
+							"cookie": [],
+							"body": "{\"message\":{\"text\":\"Location shared:\nhttps://maps.google.com/maps?q=45.5261583,-73.595346\",\"coordinates\":{\"lat\": 45.5261583,\"long\": -73.595346},\"location\":{\"address\":\"5333 avenue Casgrain\",\"name\":\"Zendesk Montreal\"},\"type\":\"location\",\"role\":\"appMaker\",\"received\":1529675742.595,\"authorId\":\"00uejytoekz520ZEG0h7\",\"avatarUrl\":\"https://media.smooch.io/5ac3da9539fe6c0022447259/icons/appicon.jpg\",\"name\":\"Galifrey\",\"_id\":\"5b2cffdec19c1a00219056c5\",\"source\":{\"type\":\"api\"}},\"conversation\":{\"unreadCount\":1,\"_id\":\"884ac26bfa519a0bb4cb0162\",\"appMakerLastRead\":1529675742.595,\"appUserLastRead\":1529675346.151}}"
```
Comment (fxlemire) [inline Smooch.postman_collection.json]:
```suggestion
							"body": "{\"message\":{\"text\":\"Location shared:\nhttps://maps.google.com/maps?q=45.5261583,-73.595346\",\"coordinates\":{\"lat\": 45.5261583,\"long\": -73.595346},\"location\":{\"address\":\"5333 avenue Casgrain\",\"name\":\"Zendesk Montréal\"},\"type\":\"location\",\"role\":\"appMaker\",\"received\":1529675742.595,\"authorId\":\"00uejytoekz520ZEG0h7\",\"avatarUrl\":\"https://media.smooch.io/5ac3da9539fe6c0022447259/icons/appicon.jpg\",\"name\":\"Galifrey\",\"_id\":\"5b2cffdec19c1a00219056c5\",\"source\":{\"type\":\"api\"}},\"conversation\":{\"unreadCount\":1,\"_id\":\"884ac26bfa519a0bb4cb0162\",\"appMakerLastRead\":1529675742.595,\"appUserLastRead\":1529675346.151}}"
```

File: Smooch.postman_collection.json [hunk 2]
```diff
@@ -10632,6 +10632,180 @@
...
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"role\": \"appMaker\",\n    \"type\": \"location\",\n    \"coordinates\": {\n        \"lat\": 45.5261583,\n        \"long\": -73.595346\n    }\n,   \"location\": {\n        \"address\": \"5333 avenue Casgrain\",\n        \"name\": \"Zendesk Montreal\"\n    }\n}"
```
Comment (fxlemire) [inline Smooch.postman_collection.json]: nit:

```suggestion
									"raw": "{\n    \"role\": \"appMaker\",\n    \"type\": \"location\",\n    \"coordinates\": {\n        \"lat\": 45.5261583,\n        \"long\": -73.595346\n    }\n,   \"location\": {\n        \"address\": \"5333 avenue Casgrain\",\n        \"name\": \"Zendesk Montréal\"\n    }\n}"
```

parrot

(reason for my nits being: I believe the `é` is part of Zendesk's official branding for the Montréal office)

**Review by fxlemire — APPROVED** [review]

**Review by fxlemire — APPROVED** [review]

**Pull request merged**